dinstaller-lib/src/dinstaller.rs

```diff
@@ -0,0 +1,46 @@
+use std::path::Path;
+use std::collections::HashMap;
+use zbus::zvariant::Value;
+use zbus::dbus_proxy;
+
+pub fn connection() -> Result<zbus::blocking::Connection, zbus::Error>{
+    let path = if Path::new("/run/d-installer/bus").exists() {
```

Comment:
In case it is useful, there is a `/run/d-installer/bus.address` file that should contain the real address.